github-orgs/kubeflow/org.yaml

```diff
@@ -1010,6 +1010,22 @@ orgs:
             privacy: closed
             repos:
               katib: write
+          wg-data-leads:
+            description: Team of Model Registry and Spark Operator
```

Comment:
Please separate Spark Operator to a different team.

Reply:
corrected